pkg/service/restore/progress.go

```diff
@@ -99,27 +98,7 @@ func (w *worker) aggregateProgress(ctx context.Context) (Progress, error) {
 
 	p.extremeToNil()
 
-	for _, v := range w.run.Views {
-		viewTableName := v.View
-		if v.Type == SecondaryIndex {
-			viewTableName += "_index"
-		}
-
-		status, err := w.client.ViewBuildStatus(ctx, v.Keyspace, viewTableName)
-		if err != nil {
-			w.logger.Error(ctx, "Couldn't get view build status",
-				"keyspace", v.Keyspace,
-				"view", v.View,
-				"error", err,
-			)
-			status = scyllaclient.StatusUnknown
-		}
-		p.Views = append(p.Views, ViewProgress{
-			View:   v,
-			Status: status,
-		})
-	}
```

Comment:
So, the reason for failing get progress of the restore task when the cluster is down, was the fact that SM was always validating build status throught the API call ? But it is not failing the call, but just logging.

Reply:
Yes, but we failed during client creation step.
Anyway, it's more consistent to never create cluster client during `sctool progress` and just rely on the data from SM DB. It also allows us for checking the progress of the previous runs, which might not be the most interesting from the view build status perspective, but we should respect it.